addons/web/static/tests/core/network/http_service.test.js

```diff
@@ -0,0 +1,44 @@
+/** @odoo-module **/
+
+import { after, expect, test } from "@odoo/hoot";
+import { mockFetch } from "@odoo/hoot-mock";
+import { get, post } from "@web/core/network/http_service";
```

Comment:
i would try to separate hoot imports from @web imports, as with hoot and its module set system, they come from completely different environments (hoot: once for all, others: specific to this file)